wiz-admission-controller/Chart.yaml

```
@@ -7,7 +7,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 3.6.0
+version: 3.6.1
```

```suggestion
version: 3.6.1-preview
```
WDYT?
Let the e2e run with that just for sanity